### DISCUSSION TITLE
Authorization failed with netlify

### DISCUSSION THREAD

**Discussion by Dvloper-Ibrahim** [community/community#37214] [New to GitHub]
I can't make netlify get access to my github repos.
After it tells me that I'm authorized, this page appears and nothing changes
![problem](https://user-images.githubusercontent.com/90285256/197784417-4d1c8fb0-c94f-4756-80ed-837437513a5b.PNG)

**Comment by cesaranzola** [comment]
I'm having the same problem.

**Comment by todovibra** [comment]
Yes I am in the same boat and noticed it a few weeks ago.
